README.md

```diff
@@ -761,9 +769,9 @@ v::consonant()->validate('xkcd'); //true
 
 See also:
 
-  * [v::alnum()](#valnum)  - a-z0-9, empty or whitespace only
-  * [v::digit()](#vdigit) - 0-9, empty or whitespace only
-  * [v::alpha()](#valpha)  - a-Z, empty or whitespace only
+  * [v::alnum()](#valnum)  - a-z0-9 or empty
+  * [v::digit()](#vdigit) - 0-9 or empty
+  * [v::alpha()](#valpha)  - a-z or empty
```

Comment:
Yes, the same file, but two kinds of changes: 
- Removing whitespace only
- Translation changes